readme.md

```diff
@@ -51,3 +51,13 @@ Default: `undefined`
 Overrides the modification timestamp for all files added to the archive.
 
 Tip: Setting it to the same value across executions enables you to create stable archives that change only when the contents of their entries change, regardless of whether those entries were "touched" or regenerated.
+
+##### buffer
+
+Type: `boolean`<br>
+Default: `true`
+
+If `true`, the resulting ZIP file contents will be a buffer. Large zip files may not be possible to buffer, depending on the size of [Buffer MAX_LENGTH](https://nodejs.org/api/buffer.html#buffer_buffer_constants_max_length).
+If `false`, the ZIP file contents will be a stream.
+
+Similar to [gulp.src's `buffer` option](https://gulpjs.com/docs/en/api/src/#options).
```

Comment:
> Yes, but it should be clearly documented in what case the option is useful.

Reply:
Hopefully clarified that now.

Reply:
It's not really clearer. As a user, I would have liked to know why the option exists when there's the `gulp.src`'s `buffer` option.

Reply:
I felt that was explained in the paragraph above. Do you want me to remove the comparison or add more detail here?

Reply:
Add more detail.